README.md

```diff
@@ -10,17 +10,16 @@ This repository contains the source code for the OpenRCA web interface.
 
 ### Build and run in Docker
 
-Build Docker image:
+To build and start `orca-ui`, `orca-proxy` and `orca-mock` containers
+simply run below commands:
 
-```bash
-$ docker build -t openrca/orca-ui .
+```sh
+cd ./utils
+docker-compose up
 ```
 
-Run Docker container:
-
-```
-$ docker run -it --name orca-ui -p 3000:3000 openrca/orca-ui
-```
+The UI is available at [http://localhost](http://localhost)
+(**IMPORTANT** using `127.0.0.1` will result in CORS violation warnings and inability to display the graph)
```

Comment:
The Mock API and proxy stuff (the residents of `utils` directory) are primarily for development purposes. I think we could move these instructions to the `Development` section below.

Also, I suggest reverting the previous instructions for `Usage` section to describe the regular usage of the UI.